tests/components/lamarzocco/test_button.py

```diff
@@ -58,7 +58,7 @@ async def test_button_error(
 
     mock_lamarzocco.start_backflush.side_effect = RequestNotSuccessful("Boom.")
 
-    with pytest.raises(HomeAssistantError, match="Button action failed."):
```

Comment:
Not sure why this was removed from the tests?

Reply:
because tests are failing, if I remove the `msg` part. What would I need to match? If I match the content from strings.json it doesn't seem to work.

Reply:
Did you update/generate the language files?

`python -m script.translations develop --all` ?

Reply:
![image](https://github.com/user-attachments/assets/b835116f-e974-4d34-b659-dfb65f4e2408)
yes. This is what the exception looks like. It would be new to me that translations are rendered at this point.

Reply:
I see your point, you could alternatively check the actual logs or assert the translation key in that raised error.

Reply:
checking for `translation_key` now. A guideline on what people should use here would be nice (also for when I'm reviewing other PRs)

Reply:
There is no guideline on this.

Nevertheless, looks good 👍 

../Frenck

Reply:
also, according to https://developers.home-assistant.io/blog/2024/03/15/exception-translations/ seems like `translation_domain` is a required argument for local logging to work then